gnomad/utils/plotting.py

```diff
@@ -215,11 +215,14 @@ def plot_multi_hail_hist(
 
         hist_source = ColumnDataSource(data)
 
+        # fmt: off
+        hide_zeros_filter = BooleanFilter(
+            [top > 0 for top in hist_source.data["top"]]  # pylint: disable=unsubscriptable-object
+        )
+        # fmt: on
```

Comment:
Had to jump through some hoops here to prevent Black from moving the comment to a different line.